static/app/views/issueDetails/streamline/occurrenceSummary.tsx

```diff
@@ -77,6 +77,15 @@ export function OccurrenceSummary({group, event, className}: OccurrenceSummaryPr
     return map;
   }, {});
 
+  if (evidenceMap?.Environment) {
+    items.push(
+      <Flex column>
+        <ItemTitle>{t('Environment')}</ItemTitle>
+        <ItemValue>{evidenceMap.Environment.value}</ItemValue>
```

Comment:
we should probably figure out some types for these names. someone is going to get these wrong

Reply:
Converted this to an enum for this file. Hopefully that makes it a little harder to mess up